public/src/css/style.css

```diff
@@ -2042,6 +2042,11 @@ ol.search_list > li::before {
     color: #555;
 }
 
+span.also a{
```

Comment:
usually it's better to avoid mentioning the tag name, `.also` is more future-proof than`span.also` and less verbose